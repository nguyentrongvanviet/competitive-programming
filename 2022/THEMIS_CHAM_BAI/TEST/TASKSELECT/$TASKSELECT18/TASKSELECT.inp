10
1 169717961
3 164800947
3 872623929
3 709009339
3 745314313
2 854776809
2 270019319
3 343101890
2 733772120
3 1000000000
