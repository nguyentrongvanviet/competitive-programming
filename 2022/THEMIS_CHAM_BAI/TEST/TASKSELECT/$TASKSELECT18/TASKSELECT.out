1
246097975
574778829
251358439
66860182
427388404
135009659
669910708
366886059
250000010
